cyberdrop_dl/crawlers/luxuretv.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, ClassVar
+
+from cyberdrop_dl.crawlers.crawler import Crawler, SupportedPaths
+from cyberdrop_dl.data_structures.url_objects import AbsoluteHttpURL
+from cyberdrop_dl.utils import css
+from cyberdrop_dl.utils.utilities import error_handling_wrapper
+
+if TYPE_CHECKING:
+    from cyberdrop_dl.data_structures.url_objects import ScrapeItem
+
+
+class Selector:
+    VIDEO_PLAYER = "video.video-js > source"
+    METADATA_SELECTOR = "script[type='application/ld+json']"
+    TITLE = "h1.title-right"
+    NEXT_PAGE = "div#pagination > a:-soup-contains('Suivante »')"
+    VIDEOS_THUMBS = "div.contents div.content a"
+
+
+PRIMARY_URL = AbsoluteHttpURL("https://luxuretv.com")
+
+
+class LuxureTVCrawler(Crawler):
+    SUPPORTED_PATHS: ClassVar[SupportedPaths] = {"Video": "/videos/...", "Search": "/searchgate/videos/<search>/..."}
+    DOMAIN: ClassVar[str] = "luxuretv"
+    FOLDER_DOMAIN: ClassVar[str] = "LuxureTV"
+    PRIMARY_URL: ClassVar[AbsoluteHttpURL] = PRIMARY_URL
+    NEXT_PAGE_SELECTOR: ClassVar[str] = Selector.NEXT_PAGE
+    _RATE_LIMIT = 3, 10
+
+    async def fetch(self, scrape_item: ScrapeItem) -> None:
+        if "searchgate" in scrape_item.url.parts:
+            return await self.search(scrape_item)
+        elif "videos" in scrape_item.url.parts:
+            video_id = scrape_item.url.name.split("-")[-1].split(".")[0]
+            return await self.video(scrape_item, video_id)
+        raise ValueError
+
+    @error_handling_wrapper
+    async def search(self, scrape_item: ScrapeItem) -> None:
+        title: str = ""
+        url = scrape_item.url.with_path(scrape_item.url.path + "/")
```

Comment:
They have bad routes handling. The trailing slash is only needed for the first page and returns 404 for any other page

Set `DEFAULT_TRIM_URLS` in the class to `False` and you should also can add a check to only add it to the first page

Reply:
The trailing slash is only added to the first page already. I've set DEFAULT_TRIM_URLS to False, but it is only used in parse_url method, not in the first page. Let me know if I misunderstood anything.

Reply:
Only add it if the URL does not end in `.html`
```suggestion
        url = scrape_item.url
        if url.name and not url.name.endswith(".html"):
            url = url / ""
```

Reply:
This is to handle the case where the input URL from the user was not the first page